pywinauto/base_wrapper.py

```diff
@@ -551,11 +551,10 @@ def verify_actionable(self):
         Raise either ElementNotEnalbed or ElementNotVisible if not
         enabled or visible respectively.
         """
-        if self.element_info.handle:
+        if self.backend.name == 'win32' and self.handle:
```

Comment:
Using backend-specific code in ```base_wrapper``` smells as a design problem. Such piece of code should be extracted to a separate method (```wait_idle```?) and moved to ```element_info``` or made private and overridden by descendant classes: ```UIAWrapper``` and ```HwndWrapper```.

Notice that there are also other methods of this class calling ```win32functions.WaitGuiThreadIdle```: ```move_mouse_input```,```mouse_move```, ```type_keys```. Are these methods bulletproof ?